<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>test_mode_panel</class>
 <widget class="QDialog" name="test_mode_panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>99</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="cb_node_1">
      <property name="text">
       <string>node 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_node_2">
      <property name="text">
       <string>node 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_node_3">
      <property name="text">
       <string>node 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_node_4">
      <property name="text">
       <string>node 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_node_5">
      <property name="text">
       <string>node 5</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>91</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="le_node_delay_1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_node_delay_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_node_delay_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_node_delay_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_node_delay_5"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_start_test">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop_test">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Test</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay Start</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>230</y>
     <width>109</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Packet Interval</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>126</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="le_packet_interval_min"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_packet_interval_max"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Packet Num</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>91</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="le_packet_num_1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_packet_num_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_packet_num_3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_packet_num_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_packet_num_5"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_check_nodes_status">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>250</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Modem</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>101</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="lab_node_status_1">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_node_status_2">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_node_status_3">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_node_status_4">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_node_status_5">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Modem Status</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_show_log">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Log</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
